app/controllers/cancelCase/AnyOtherSupportingCancellationDocsController.scala

```diff
@@ -54,16 +55,30 @@ class AnyOtherSupportingCancellationDocsController @Inject()(identify: Identifie
       formWithErrors => Future.successful(BadRequest(view(formWithErrors, backLink))),
       value =>
         for {
-          updatedAnswers <- Future.fromTry(request.userAnswers.set(AnyOtherSupportingDocsPage, value))
-          _ <- sessionRepository.set(updatedAnswers)
+          answersWithSupportingDocumentation <- Future.fromTry(request.userAnswers.set(AnyOtherSupportingDocsPage, value))
```

Comment:
not related to this ticket, but we might need to add similar logic here
https://github.com/hmrc/import-voluntary-disclosure-frontend/blob/master/app/controllers/AnyOtherSupportingDocsController.scala